index.js

```diff
@@ -240,6 +242,7 @@ async function run() {
   try {
     // Get inputs
     const accessKeyId = core.getInput('aws-access-key-id', { required: false });
+    const audience = core.getInput('audience', { required: false }) || DEFAULT_AUDIENCE;
```

Comment:
```suggestion
    const audience = core.getInput('audience', { required: false });
```

Reply:
Wouldn't we need to change the `required:` to `true` here then, even if we defaulted it in the action.yml?

Reply:
No - because when  you provide a default, then its no longer "required". so behind the scenes, the logic you have here is what the action is doing from that config